packages/ui/package.json

```diff
@@ -3,45 +3,53 @@
 	"version": "0.0.1",
 	"private": true,
 	"scripts": {
-		"dev": "vite dev",
-		"build": "vite build",
-		"preview": "vite preview",
-		"test": "playwright test",
+		"dev": "npm run protos && vite dev",
+		"build": "npm run protos && vite build",
+		"preview": "npm run protos && vite preview",
+		"test": "npm run protos && playwright test",
 		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
 		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
 		"lint": "prettier --plugin-search-dir . --check . && eslint .",
 		"format": "prettier --plugin-search-dir . --write . && eslint . --fix",
 		"waku:start": "docker-compose -f ./tests/docker-compose.yaml up -d",
-		"waku:stop": "docker kill tests-waku-1"
+		"waku:stop": "docker kill tests-waku-1",
+		"protos": "protons ./src/lib/protos/*.proto"
 	},
 	"devDependencies": {
 		"@fontsource/source-code-pro": "^4.5.14",
 		"@fontsource/source-sans-pro": "^4.5.11",
 		"@fontsource/source-serif-pro": "^4.5.9",
-		"@playwright/test": "1.29.2",
+		"@playwright/test": "1.30.0",
 		"@rollup/plugin-inject": "^5.0.3",
-		"@semaphore-protocol/group": "3.0.0-beta.7",
-		"@semaphore-protocol/identity": "3.0.0-beta.7",
-		"@semaphore-protocol/proof": "3.0.0-beta.7",
-		"@sveltejs/adapter-static": "^1.0.5",
-		"@sveltejs/kit": "^1.2.6",
-		"@typescript-eslint/eslint-plugin": "^5.49.0",
-		"@typescript-eslint/parser": "^5.49.0",
+		"@semaphore-protocol/group": "3.0.0",
+		"@semaphore-protocol/identity": "3.0.0",
+		"@semaphore-protocol/proof": "3.0.0",
+		"@sveltejs/adapter-static": "^1.0.6",
+		"@sveltejs/kit": "^1.3.10",
+		"@typescript-eslint/eslint-plugin": "^5.50.0",
+		"@typescript-eslint/parser": "^5.50.0",
 		"big-integer": "^1.6.51",
-		"eslint": "^8.32.0",
+		"eslint": "^8.33.0",
 		"eslint-config-prettier": "^8.6.0",
 		"eslint-plugin-svelte3": "^4.0.0",
 		"ethers": "^5.7.2",
 		"node-stdlib-browser": "^1.2.0",
 		"prettier": "^2.8.3",
 		"prettier-plugin-svelte": "^2.9.0",
-		"sass": "^1.57.1",
+		"protons": "^7.0.0",
+		"sass": "^1.58.0",
 		"svelte": "^3.55.1",
-		"svelte-check": "^3.0.2",
+		"svelte-check": "^3.0.3",
 		"svelte-preprocess": "^5.0.1",
-		"tslib": "^2.4.1",
-		"typescript": "^4.9.4",
-		"vite": "^4.0.4"
+		"tslib": "^2.5.0",
+		"typescript": "^4.9.5",
+		"vite": "^4.1.1"
 	},
-	"type": "module"
+	"type": "module",
+	"dependencies": {
+		"bigint-conversion": "^2.3.0",
+		"js-waku": "^0.30.0",
+		"multiaddr": "^10.0.1",
+		"protons-runtime": "^5.0.0"
+	}
```

Comment:
With svelte everything is `devDependecy` because svelte is actually a compiler so it crawls through all the imports, and bundles only what is used at compile time.